js/widgets/sampler.js

```diff
@@ -380,14 +370,13 @@ function SampleWidget() {
                 // eslint-disable-next-line no-unused-vars
                 reader.onload = function (event) {
                     // if the file is of .wav type, save it
-                    if (reader.result.substring(reader.result.indexOf(":")+1, reader.result.indexOf(";")) === "audio/wav") {                        
+                    if (reader.result.substring(reader.result.indexOf(":")+1, reader.result.indexOf(";")) === "audio/wav") {
                         if (reader.result.length <= 1333333) {
                             that.sampleData = reader.result;
                             that.sampleName = fileChooser.files[0].name;
                             that._addSample();
                         } else {
-                            that.sampleData = reader.result;
-                            that.getSampleLength();
+                            that.activity.errorMsg(_("Warning: Sample is bigger than 1MB."), that.timbreBlock);
```

Comment:
Maybe we need to update this message as well, since we are not actually loading the sample.

"Your sample cannot be loaded because it is >1MB."  ???

Reply:
Yes, it should also be updated.